conf/sample.config.inc.php

```diff
@@ -33,11 +33,11 @@
  *
  */
 $conf['datasources']['localhost'] = array(
-	'host'	=> 'localhost',
+	'host'	=> getenv('ANEMOMETER_MYSQL_1_PORT_3306_TCP_ADDR') ?: 'localhost',
```

Comment:
if your folder names changes, would the environment variables still be valid?

I think that name comes from `<FolderName>_<ImageName>_<Container#>_PORT_3306_TCP_ADDR`